### PR TITLE
feat(autoware_adapi_v1_msgs): add heartbeat

### PR DESCRIPTION
**Pull request by isamu-takagi**
## Description

Add heartbeat message. This feature is requested by https://github.com/orgs/autowarefoundation/discussions/4561

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by xmfcx** [comment]
If this is related, could you add it to the PR description?
- https://github.com/orgs/autowarefoundation/discussions/4561

**Comment by xmfcx** [comment]
https://github.com/autowarefoundation/autoware.core/blob/5ab56d1b560dfce85fb7d272e4ecb42022af32cc/common/autoware_control_center_msgs/msg/Heartbeat.msg#L1-L10

Maybe we could also add a similar `uint16 sequence_number 0` for the listener to detect if a heartbeat was missed.

Related PR:
- https://github.com/autowarefoundation/autoware.core/pull/73

File: autoware_adapi_v1_msgs/system/msg/Heartbeat.msg [hunk 1]
```diff
@@ -0,0 +1 @@
+builtin_interfaces/Time stamp
```
Comment (yukkysaito) [inline autoware_adapi_v1_msgs/system/msg/Heartbeat.msg:1]: Do you not need a counter?

Comment (xmfcx) [reply autoware_adapi_v1_msgs/system/msg/Heartbeat.msg:1]: @yukkysaito I've also suggested the same idea [above](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/50#issuecomment-2022460258)

Comment (isamu-takagi) [reply autoware_adapi_v1_msgs/system/msg/Heartbeat.msg:1]: Thank you. Add a message counter https://github.com/autowarefoundation/autoware_adapi_msgs/pull/50/commits/cd39fdd0260c3b2e3090a785b411fe1c99302725.

File: autoware_adapi_v1_msgs/system/msg/Heartbeat.msg [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+uint16 seq
```
Comment (xmfcx) [inline autoware_adapi_v1_msgs/system/msg/Heartbeat.msg:2]:
```suggestion

# Sequence number for order verification, wraps at 65535.
uint16 seq
```

Maybe a short comment like this to show the intent?

**Review by xmfcx — APPROVED** [review]

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**